pom.xml

```diff
@@ -212,11 +233,20 @@
     </dependency>
 
     <!-- plugin dependencies -->
+    <dependency>
+      <groupId>org.jenkins-ci.plugins</groupId>
+      <artifactId>plain-credentials</artifactId>
+      <version>1.5</version>
```

Comment:
Yes, this is required to be able to configure the credentials, at least during the CasC tests. I will see if I can restrict the scope to tests as well without breaking manual configuration with the Google Robot Private Key credentials

Reply:
It sounds like a bug of JCasC.
It’s strange that plugins require dependencies that aren’t necessary for plugin features.
Please consider to file a Jira ticket for JCasC.

Reply:
These dependencies need to be limited to test and are needed because JCasC does not do plugin installation.

Reply:
Should not be needed if the credential plugin had direct JCasC support

Reply:
I discovered that I was able to scope to tests by also marking optional as true. This is so that hpi does not include it, like this example from the parent plugin pom [here](https://github.com/jenkinsci/plugin-pom/blob/master/pom.xml#L290).

The reason is that the hpi plugin includes test dependencies by default: https://jenkinsci.github.io/maven-hpi-plugin/run-mojo.html#dependencyResolution